gen.py

```diff
@@ -503,6 +503,9 @@ def codelists_to_docs(lang):
         if not fname.endswith('.json'): continue
         with open(json_file) as fp:
             codelist_json = json.load(fp)
+            for i, x in enumerate(codelist_json['data']):
+                    if ('status' in x and x['status'] != 'active'):
+                        codelist_json['data'][i]['code'] = codelist_json['data'][i]['code'] + " (withdrawn)"
```

Comment:
This looks like it amends withdrawn codes in order to display them differently. I wonder if would be better to do this in the template.

It would save a loop (since there’s already a loop that happens in the template) and might look neater. Also, I guess changing codelist codes like this could lead to surprise consequences – I don’t know.

Reply:
Yeah, seems sensible to do it in the template instead, although I recall @Ocre42 mentioning something being a bit of a nag to deal with in the template (can't recall the full conversation, so deferring to him here). If that wasn't the case, doing it on the template should be the preferred implementation.

Reply:
I've followed your suggestion but github is still having some issues updating requests, thanks for the suggestion @andylolz